static/components/around-cards.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+// Component
+import PoiCard from './PoiCard'
+
+class App extends React.Component {
```

Comment:
Considering [the React file name conventions](https://gist.github.com/koistya/d7a507438c741ee6adb5) again, the file should be called _App.js_, or the class would be _AroundCards.js_ (and the filename should be adapted in CamelCase)